mesonbuild/cargo/builder.py

```diff
@@ -44,7 +44,7 @@ def string(self, value: str) -> mparser.StringNode:
         :param value: the value of the string
         :return: A StringNode
         """
-        return mparser.StringNode(self._token('string', value))
+        return mparser.StringNode(self._token('string', value), escape=False)
```

Comment:
The commit message says "cargo: escape strings passed to the builder" but you mean "do not escape"? My understanding is that the generated AST can use windows path sep instead of unix. Maybe instead of having escape=False everywhere, we should just ensure to convert to unix places we expect a path?

Reply:
Ok make sense.